packages/cli/src/models/compiler/Compiler.ts

```diff
@@ -1,42 +1,18 @@
 import { exec } from 'child_process';
 import { FileSystem, Contracts, Logger } from 'zos-lib';
-import Truffle from '../initializer/truffle/Truffle';
-import SolidityProjectCompiler from './solidity/SolidityProjectCompiler';
-import { CompilerOptions } from './solidity/SolidityContractsCompiler';
 
 const log = new Logger('Compiler');
 
-export default {
+const Compiler = {
   async call(): Promise<{ stdout: string, stderr: string } | void> {
```

Comment:
nit: `call()` shouldn't have the `void` return type. It was added for the particular case in which we compile with `solc` in `compileWithSolc`, but this function has been removed in this same PR.